app/main.py

```diff
@@ -1,3 +1,19 @@
 def cache(func):
-    # Write your code here
-    pass
+
+    stores_results = {}
+
+    def wrapper(*args):
+
+        if args in stores_results:
+
+            print("Getting from cache")
+            return stores_results[args]
+
+        else:
+
+            stores_results[args] = func(*args)
+            print("Calculating new result")
+
+            return stores_results[args]
```

Comment:
you have a duplicate `return`, so take it outside the if-else statement; try to rewrite your decorator not to have `return` duplicated